lib/seed_helper.rb

```diff
@@ -58,5 +58,29 @@ def create_role(role_name, permission_names, builtin, check_audit = true)
       permissions = Permission.where(:name => permission_names)
       create_filters(role, permissions)
     end
+
+    def rename_existing(role, original_name)
```

Comment:
SeedHelper is generic helper for seeding, while this and `rename_with_free_name` methods are Role specific, their names should mention role in name at least.